echopype/calibrate/calibrate_ek.py

```diff
@@ -879,22 +909,23 @@ def _compute_cal(self, cal_type, waveform_mode, encode_mode) -> xr.Dataset:
         # Raise error when waveform_mode and actual recording mode do not match
         # This simple check is only possible for BB-only data,
         #   since for data with both BB and CW complex samples,
-        #   frequency_start will exist in echodata.beam for the BB channels
-        if waveform_mode == "BB" and "frequency_start" not in self.echodata.beam:
+        #   frequency_start will exist in echodata["Sonar/Beam_group1"] for the BB channels
+        if waveform_mode == "BB" and "frequency_start" not in self.echodata["Sonar/Beam_group1"]:
             raise ValueError("waveform_mode='BB' but broadband data not found!")
 
         # Set use_beam_power
-        #  - True: use self.echodata.beam_power for cal
-        #  - False: use self.echodata.beam for cal
+        #  - True: use self.echodata["Sonar/Beam_group2"] for cal
+        #  - False: use self.echodata["Sonar/Beam_group1"] for cal
         use_beam_power = False
 
         # Warn user about additional data in the raw file if another type exists
         # When both power and complex samples exist:
-        #   complex samples will be stored in echodata.beam
-        #   power samples will be stored in echodata.beam_power
+        #   complex samples will be stored in echodata["Sonar/Beam_group1"]
+        #   power samples will be stored in echodata["Sonar/Beam_group2"]
         # When only one type of samples exist,
-        #   all samples with be stored in echodata.beam
-        if self.echodata.beam_power is not None:  # both power and complex samples exist
+        #   all samples with be stored in echodata["Sonar/Beam_group1"]
+        try:
+            self.echodata["Sonar/Beam_group2"]
```

Comment:
would this line
```
self.echodata["Sonar/Beam_group2"]
```
result in something being printed out? I think we don't want that behavior?

Reply:
See https://github.com/OSOceanAcoustics/echopype/pull/762#discussion_r941612868.. I think a lot of this `try/except` is kind of ugly haha

Reply:
Removed and revert back to checking of `self.echodata["Sonar/Beam_group2"] is None` since now groups that are not in echodata returns None when trying to be accessed.